sandbox-template.md

```diff
@@ -0,0 +1,162 @@
+# CNCF SIG Sandbox Project Request Template & Promotion Template
+
+
+# <Project> 
+
+
+## <Promotion/Acceptance> to CNCF <Incubation/Sandbox>
+
+**Primary aim: This template helps a project to prepare for a presentation to the SIG, and then helps the SIG share the collated information with the TOC.**
+
+**Authors:**
+
+
+# Background
+
+**_Link to TOC PR_**
+
+**_Link to Presentation_**
+
+**_Link to GitHub project_**
+
+
+## Project Goal
+
+
+## Current Status
+
+
+## Future Plans
+
+
+# Project Scope
+
+
+## Clear project definition
+
+**_Does the project have a clear and well defined scope_**
+
+## Value-add to the CNCF ecosystem
+
+**_Does the project have a clear value add to the current project ecosystem. How does it relate to other projects with overlapping capabilities._**
+
+
+## Alignment with other CNCF projects
+
+**_Does the project align and actively collaborate with other CNCF projects?_**
+
+**_Does the project require any specific versions of projects (or APIs) to interoperate?  (e.g. K8s API, CSI, CNI, CRI)?_**
+
+**_Does the project augment or benefit other CNCF projects?_**
+
+## Anticipated use cases
+
+**_What problems does the project solve?_**
+
+## Alignment with SIG Reference Model
+
+**_Does the project align with the SIG CNCF reference model and which capabilities does it require/provide at each level of the reference model._**
+
+
+## High level architecture
+
+**_Describe the overall architecture of the project. Feel free to add diagrams._**
+
+# Formal Requirements
+
+**_Document that the project fulfills the requirements as documented in the [CNCF graduation criteria for sandbox](https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc#sandbox-stage)_**
+
+**_Are there any anticipated issues with any of the criteria ?_**
+
+
+# CNCF IP Policy
+
+**_Becoming a sandbox project requires adoption of the CNCF IP Policy: [https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy](https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy)_**
+
+**_Note: there is a grace period after becoming a sandbox period to enable projects to adopt the policy, however, some prep is required to ensure there are no major blockers._**
+
+**_Has the IP policy been reviewed?_**
+
+**_List the repos for the project and their current license_**
+
+**_List any dependent repos (upstream/downstream) that are required to build the project  (including but not limited to  libraries, commercial tools, plugins)_**
+
+**_What actions are required to be compliant with the IP policy?_**
+
+
+# Other Considerations
+
+## **_Please note, these are not gating criteria but rather to:_**
+
+
```

Comment:
This section seems like good guidance to a SIG chair (or their delegate) who is reviewing the project and preparing information for TOC recommendation, but doesn't seem link it belongs in template.  (Maybe move to top and include in a section which is "guidance: to be removed when using this template"